stwcs/updatewcs/corrections.py

```diff
@@ -70,14 +70,20 @@ def updateWCS(cls, ext_wcs, ref_wcs):
         ext_wcs.idcmodel.ocx = copy.deepcopy(ext_wcs.idcmodel.cx)
         ext_wcs.idcmodel.ocy = copy.deepcopy(ext_wcs.idcmodel.cy)
 
-        newkw = {'TDDALPHA': None, 'TDDBETA': None,
-                 'OCX10': ext_wcs.idcmodel.ocx[1, 0],
-                 'OCX11': ext_wcs.idcmodel.ocx[1, 1],
-                 'OCY10': ext_wcs.idcmodel.ocy[1, 0],
-                 'OCY11': ext_wcs.idcmodel.ocy[1, 1],
-                 'TDD_CTA': None, 'TDD_CTB': None,
-                 'TDD_CYA': None, 'TDD_CYB': None,
-                 'TDD_CXA': None, 'TDD_CXB': None
+        ocxy_comment = "linear term without scale from IDCTAB file"
```

Comment:
If I recall correctly at least the two of the OCX/OCY keywords represent a scale in the IDC distortion file. I may be misremembering but it's good to verify this.

Reply:
I based this comment on description from https://www.stsci.edu/itt/review/DrizzlePac/HTML/ch34.html

However, according to @stsci-hack: